src/oscar/apps/basket/forms.py

```diff
@@ -299,17 +299,19 @@ def cleaned_options(self):
         return options
 
 
-class SimpleAddToBasketForm(AddToBasketForm):
+class SimpleAddToBasketMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if 'quantity' in self.fields:
+            self.fields['quantity'].initial = 1
+            self.fields['quantity'].widget = forms.HiddenInput()
+
+
+class SimpleAddToBasketForm(SimpleAddToBasketMixin, AddToBasketForm):
     """
     Simplified version of the add to basket form where the quantity is
     defaulted to 1 and rendered in a hidden widget
 
     Most of the time, you won't need to override this class. Just change
```

Comment:
This docstring is not correct - simply changing `AddToBasketForm` does not work and never has. I think we need to replace this to explain how to override this - i.e., to explain that if you have changed `AddToBasketForm` then you need to redefine `SimpleAddToBasketForm` as well, by doing:

```
class SimpleAddToBasketForm(SimpleAddToBasketMixin, AddToBasketForm):
    pass
```

Reply:
Ok done.